sound/soc/codecs/hdac_hdmi.c

```diff
@@ -1176,7 +1176,7 @@ static int hdac_hdmi_add_cvt(struct hdac_device *hdev, hda_nid_t nid)
 	struct hdac_hdmi_cvt *cvt;
 	char name[NAME_SIZE];
 
-	cvt = kzalloc(sizeof(*cvt), GFP_KERNEL);
+	cvt = devm_kzalloc(&hdev->dev, sizeof(*cvt), GFP_KERNEL);
 	if (!cvt)
 		return -ENOMEM;
 
```

Comment:
cvt->name = devm_kstrdup(&hdev->dev, name, GFP_KERNEL); at line 1185?

Reply:
Sorry, what is the question here? I don't see the issue.